src/code/HTTPRequestHandler/HTTPRequestHandler.cpp

```diff
@@ -64,6 +64,12 @@ void HTTPRequestHandler::POST(HTTPCall &request)
 	{
 		const IPath *root(request.getLocation()->getRoot());
 		Path url(root->get() + request.getBasicRequest().getPath());
+		if (httpRequestHandlerPOST::isFileUpload(request))
+		{
+			FileManager::createFile(request.getBasicRequest(), root->get());
+			ResponseHeader response(HTTPStatusCode(HTTPStatusCode::CREATED), request.getLocation()->getErrorPageSet());
+			return (request.setResponse(response.getResponse()));
```

Comment:
i think it is unclear to use `return` this way.
since we return `void` then maybe have `return;` and not `return (value);`